deeplake/core/tensor.py

```diff
@@ -1733,6 +1733,34 @@ def create_vdb_index(
         distance: Union[DistanceType, str] = DistanceType.L2_NORM,
         additional_params: Optional[Dict[str, int]] = None,
     ):
+        """
+        Create similarity search index for embedding tensor.
+
+        Args:
+            id (str): Unique identifier for the index.
```

Comment:
@azat-manukyan We should probably modify this api so the default `id` is "hsnw_1", because right now, that's the only hard coded name that works in the search.... I think. @Sounakal